profiles/package.mask

```diff
@@ -33,6 +33,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Leonardo Hernández Hernández <leohdz172@proton.me> (2023-09-09)
+# This package has been replaced by sys-apps/eza.
```

Comment:
This should say why it has been replaced. For example: Unmaintained upstream since X years. An actively developed alternative is available as sys-apps/eza.

Reply:
Done